src/devices/Sgp30/README.md

```diff
@@ -0,0 +1,45 @@
+﻿# Sensorion SGP30 TVOC + eCO2 Gas Sensor
+This is a device binding for the Sensorion SGP30 TVOC + eCO2 Gas Sensor.
+
+## Datasheet
+[SGP30](https://cdn.shopify.com/s/files/1/0174/1800/files/Sensirion_Gas_Sensors_SGP30_Datasheet_EN-1148053.pdf)
+
+## Binding Notes
+**NOTE**: Once initialised, this sensor should be queried at least once per second. If more than several seconds pass,
+the initialisation routine should be called again to ensure the device is returning valid data.
+
+```cs
```

Comment:
`csharp` will render better. Github is quite tolerant but an other site or IDe may not be